lib/views/options.html

```diff
@@ -157,13 +157,13 @@
                   </p>
                   <small>
                     <span class="label label-danger">Note:</span>
-                    <span style = "color: rgba(75, 174, 79, 1.00);">Green, </span>
-                    <span style = "color: rgba(164, 59, 64, 1.00);">Mexican Red</span>
-                    and <span style = "color: rgba(243, 66, 53, 1.00);">Red</span>
+                    <span style = "color: rgba(86, 179, 90, 1.00);">#4BAE4F, </span>
```

Comment:
I find it a bit strange that the usdr is supposed to insert colors in rgba, but here we refer to the colors by their hexadecimal numbers... Maybe we could uniformize this?